chart/infra-server/static/flavors.yaml

```diff
@@ -771,6 +771,28 @@
       value: "4"
       kind: optional
 
+    - name: aws-region
+      description: aws region for cluster
+      value: "us-east-2"
+      kind: optional
+
+    - name: managed-cp
+      description: Use managed control-plane?
+      value: "false"
+      kind: optional
+      help: |
+        Run as ROSA HCP?
```

Comment:
Should this parameter be exposed to the user?

Reply:
Yes, I think it is confusing now, because you could set `Use Managed Control Plane` to `true` in classic ROSA - what would the effect be?